app/views/submission_notifications.scala.html

```diff
@@ -17,33 +17,42 @@
 @import java.time.format.DateTimeFormatter
 
 @import models.declaration.notifications.Notification
-@import models.declaration.submissions.SubmissionStatus
+@import models.declaration.submissions.{Submission, SubmissionStatus}
 
 @this(main_template: views.html.main_template)
 
-@(notifications: Seq[Notification])(implicit request: Request[_], messages: Messages)
+@(submission: Submission, notifications: Seq[Notification])(implicit request: Request[_], messages: Messages)
 
 @main_template(
-    title = messages("notifications.title")
+    title = messages("notifications.title"),
+    fullWidth = true
 ) {
 
     @components.back_link("/customs-declare-exports/submissions")
 
     <h1>@messages("notifications.title")</h1>
 
-    <div>
-        <table>
+    <table>
+        <thead>
             <tr class="table-row table-head">
                 <th class="table-cell">@messages("notifications.status")</th>
                 <th class="table-cell">@messages("notifications.dateAndTime")</th>
+                <th class="table-cell" colspan="2"></th>
             </tr>
-
-            @for(notification <- notifications){
-                <tr class="table-row">
-                    <td class="table-cell">@SubmissionStatus.retrieve(notification.functionCode, notification.nameCode)</td>
-                    <td class="table-cell">@DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm").format(notification.dateTimeIssued)</td>
-                </tr>
-            }
-        </table>
-    </div>
+        </thead>
+
+        <tbody>
+        @for(notification <- notifications){
+            <tr class="table-row">
+                <td class="table-cell">@notification.status.toString</td>
+                <td class="table-cell">@DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm").format(notification.dateTimeIssued)</td>
```

Comment:
Please do not build formatter at each request.

Reply:
Sorted